cgi-bin/LJ/Widget/Location.pm

```diff
@@ -265,6 +265,7 @@ sub country_options {
     my %countries;
     # load country codes
     DW::Countries->load( \%countries );
+    delete %countries->{UK};
```

Comment:
The syntax for this reference needs to be `$countries{UK}` instead of `%countries->{UK}`, which produces the following warning: `Using a hash as a reference is deprecated at cgi-bin/LJ/Widget/Location.pm line 268.`  But apart from that, it is generally doing the right thing in the right place. :heart: